bot/plugins/simple.py

```diff
@@ -79,6 +79,14 @@
     ('!lurk', 'thanks for lurking, {user}!'),
     ('!ohai', 'ohai, {user}!'),
     ('!playlist', 'HearWeGo: https://www.youtube.com/playlist?list=PL44UysF4ZQ23B_ITIqM8Fqt1UXgsA9yD6'),  # noqa: E501
+    (
+        '!puzzles',
+        'I have made a series of puzzles for people to attempt. '
+        'Here is a link to the youtube playlist: '
+        'https://www.youtube.com/playlist?list=PLWBKAf81pmOZdQZfVoWFPu0GyvGwmCWNr. '  # noqa: E501
+        'Here is a link to the GitHub repo with solutions: '
+        'https://github.com/anthonywritescode/puzzles. ',
+    ),
```

Comment:
this should use the playlist lookup similar to the irc bot, hmmm maybe I just need to add it to the config

Reply:
I totally misunderstood how this code worked 🙃 -- I ended up copy pasting to make this work on stream -- sorry for giving you the roundabout!